### DISCUSSION TITLE
Cannot connect to the sharepoint server 2019 site with Connect-PnPOnline

### DISCUSSION THREAD

**Discussion by jpmvasquez** [pnp/powershell#4001] [Q&A]
Hi,

I'm working with SharePoint Server 2019 and want to use the PnP.Powershell cmdlets for classic to modern pages modernization purpose.

I use the PnP.PowerShell version 2.4.0 and I have powershell 7

I use this command to connect to my site : 
Connect-PnPOnline -Url "https://..../sites/..." -TransformationOnPrem -CurrentCredentials -ReturnConnection -Verbose 
-ValidateConnection
Executing this command gives me a 403 FORBIDDEN.

Note that I execute this command directly in the frontal server of the sharepoint

Can someone please give me some guidance on how I can connect to this SP 2019 on-premise ?

Thanks,
Regards

**Comment by waaromikniet** [comment]
PnP.PowerShell only works against online I think.

You should use the older version
https://www.powershellgallery.com/packages/SharePointPnPPowerShell2019/3.6.1902.2
